tests/ia/pages/alertsPage.js

```diff
@@ -108,4 +114,11 @@ module.exports = {
       assert.equal(warningColor, this.colors.warning, 'Warning alert is silence');
     }
   },
+
+  checkContainingLabels(expectedLabels, alertName = null) {
```

Comment:
please change this method to accept object with keys: `primaryLabels`, `secondaryLabels` and `alertName`